composer.json

```diff
@@ -11,16 +11,22 @@
         }
     ],
     "require": {
-        "silverstripe/framework": "^4.0@dev",
-        "silverstripe/cms": "^4.0@dev"
+        "silverstripe/framework": "~4.0",
+        "silverstripe/cms": "~4.0",
+        "silverstripe/asset-admin": "~1.0.0"
```

Comment:
We prefer semver `^1.0` vs `~1.0.0`. Note that `^1.0` and `~1.0` are equivalent, but the former is preferred.